sdk/eventhub/event-hubs/samples/sendEvents.ts

```diff
@@ -42,18 +40,36 @@ async function main(): Promise<void> {
     //   { body: { "message": "Hello World 2" } },
     //   { body: { "message": "Hello World 3" } }
     // ];
+    console.log("Sending single event...");
+    const scientist = listOfScientists[0];
+    producer.send({ body: `${scientist.firstName} ${scientist.name}` });
+
+    console.log("Sending batch events...");
+    const events: EventData[] = [];
     for (let index = 0; index < listOfScientists.length; index++) {
       const scientist = listOfScientists[index];
       events.push({ body: `${scientist.firstName} ${scientist.name}` });
     }
-    console.log("Sending batch events...");
-
     await producer.send(events);
+
+    console.log("Creating and sending batch events...");
+    const eventDatabatch = await producer.createBatch();
+    for (let index = 0; index < listOfScientists.length; index++) {
+      const scientist = listOfScientists[index];
+      const isAdded = eventDatabatch.tryAdd({ body: `${scientist.firstName} ${scientist.name}` });
+      if (!isAdded) {
+        console.log(`Unable to add event ${index} to the batch`);
+        break;
+      }
+    }
+    await producer.send(eventDatabatch);
```

Comment:
Preview 1 users who might copy this sample file and try to run it will get errors as createBatch is not supported in Preview 1

What about commenting this section for batch with a comment on top if it saying that `Below variation of send will be available in the upcoming release`?